code/__DEFINES/dcs/signals.dm

```diff
@@ -482,13 +482,20 @@
 ///from [/obj/structure/closet/supplypod/proc/endlaunch]:
 #define COMSIG_SUPPLYPOD_LANDED "supplypodgoboom"
 
+// /obj signals for economy
+///called when the payment component tries to charge an account.
+#define COMSIG_OBJ_ATTEMPT_CHARGE "obj_attempt_simple_charge"
+	#define COMSIG_ITEM_CANCEL_CHARGE  (1<<0)
+///Called when a payment component changes value
+#define COMSIG_OBJ_ATTEMPT_CHARGE_CHANGE "obj_attempt_simple_charge_change"
```

Comment:
This doesn't seem to be ever used. What's that about?

Reply:
This is more to set up a framework for replacing existing machinery that you _can charge_ people with, which includes machinery with variable prices.